### PR TITLE
FOUR-10670: Designer Asset libraries: Complete the quick access

### PR DESCRIPTION
**Pull request by CarliPinell**
## Solution
All quick access to Assets were implemented in this ticket. The screen behavior after open each quick access button was not specified, but a generic way was applied to open modals in order to reuse existent code in PM4 Core and Packages.

## How to Test
User must have installed Decision-engines, Collections and Data-Sources packages.

## Related Tickets & Packages
- FOUR-10670 package-data-sources
- FOUR-10670 package-collections
- FOUR-10670 package-decision-engines

## Code Review Checklist
- [ ] I have pulled this code locally and tested it on my instance, along with any associated packages.
- [ ] This code adheres to [ProcessMaker Coding Guidelines](https://github.com/ProcessMaker/processmaker/wiki/Coding-Guidelines).
- [ ] This code includes a unit test or an E2E test that tests its functionality, or is covered by an existing test.
- [ ] This solution fixes the bug reported in the original ticket.
- [ ] This solution does not alter the expected output of a component in a way that would break existing Processes.
- [ ] This solution does not implement any breaking changes that would invalidate documentation or cause existing Processes to fail.
- [ ] This solution has been tested with enterprise packages that rely on its functionality and does not introduce bugs in those packages.
- [ ] This code does not duplicate functionality that already exists in the framework or in ProcessMaker.
- [ ] This ticket conforms to the PRD associated with this part of ProcessMaker.
ci:deploy
ci:next

### CODE REVIEW

File: resources/js/processes/designer/AssetButtons.vue [hunk 1]
```diff
@@ -1,10 +1,16 @@
 <template>
   <b-card-text>
     <div class="button-container">
-      <slot
-        name="select-template-slot"
-        class="custom-slot"
-      ></slot>
+      <b-button
+        variant="primary"
```
Comment (fagubla) [inline resources/js/processes/designer/AssetButtons.vue:5]: remove this line, the color is set in the CSS

**Comment by tokensPM** [comment]
QA server was successfully deployed https://ci-55ffefade6.eng.processmaker.net

**Review by fagubla — CHANGES_REQUESTED** [review]

File: resources/js/processes/designer/AssetButtons.vue (hunk 1 above)
Comment (CarliPinell) [reply resources/js/processes/designer/AssetButtons.vue:5]: Solved

**Comment by tokensPM** [comment]
QA server was successfully deployed https://ci-55ffefade6.eng.processmaker.net

**Review by fagubla — APPROVED** [review]

**Pull request merged**